### PR TITLE
CASMUSER-3122: Add unfied guide md files, refactor docs-product-manifest

### PR DESCRIPTION
**Pull request by keopp-hpe**
## Summary and Scope

This PR adds the unified admin and install guide markdown files and corrects the docs-product-manifest.yaml file to point to them from the product root.

## Issues and Related PRs

* Resolves [CASMUSER-3122](https://jira-pro.its.hpecorp.net:8443/browse/CASMUSER-3122)

### Test description:

Will be tested when IUF unified docs are tested.

## Risks and Mitigations

Low risk

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [X] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Comment by alanm-hpe** [comment]
How are these guides supposed to be navigated? I would have expected some sort of intro and table of contents. What's the plan to deduplicate all of the guides that make up these two new admin and install guides?

**Comment by keopp-hpe** [comment]
> How are these guides supposed to be navigated? I would have expected some sort of intro and table of contents. What's the plan to deduplicate all of the guides that make up these two new admin and install guides?

These new files will be gathered by the IUF Unified docs tool into a single "book" of docs.  The navigation to them will be provided by the IUF tooling that gathers and generates the final document.  The instruction from IUF was to render the current guides as single md documents to be pulled into the unified doc of all products.

The current guides that we already deliver will still be produced and available.

**Comment by keopp-hpe** [comment]
I will have one more commit coming here.  One to make the build script that produces all the pubs docs able to selectively produce just the single file guides.

**Review by alanm-hpe — APPROVED** [review]

File: release.sh [hunk 1]
```diff
@@ -173,6 +179,7 @@ setup_nexus_repos
 sync_repo_content
 sync_image_content
 iuf-validate "${BUILDDIR}/iuf-product-manifest.yaml"
+build_docs
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply release.sh:182]: Found a typo in the function name for build_docs.  Fixed, squashed, pushed.

**Comment by keopp-hpe** [comment]
Build is now failing because the "dita" command is not on the build server/container.

File: Makefile [hunk 2]
```diff
@@ -23,16 +23,23 @@
...
+all : check_env build_doc_tools_image generate_release
 
 check_env:
-ifndef ARTIFACTORY_USER
-        $(error ARTIFACTORY_USER is undefined)
-endif
-ifndef ARTIFACTORY_TOKEN
-        $(error ARTIFACTORY_TOKEN is undefined)
-endif
+#ifndef ARTIFACTORY_USER
```
Comment (alanm-hpe) [inline Makefile:33]: Could we leave this in? It is useful when building the release to know what needs to be defined.

Comment (keopp-hpe) [reply Makefile]: I had commented them out for local testing.  Forgot to put them back in.  Good catch!

**Pull request merged**